Source/items.h

```diff
@@ -348,6 +349,20 @@ struct ItemStruct {
 			return false;
 		}
 	}
+
+	UiFlags getTextColor() const

```

Comment:
Can you add the doxygen comments for the method?

Reply:
Can you provide some text for the comment? Not sure what you want to be clarified by a comment.

Reply:
I was under the impression we were adding comments even to obvious method for consistency/convention. If not, feel free to disregard.

Reply:
Not to obvious stuff I think. Though that can be subjective.